packages/analytics-js/src/components/userSessionManager/UserSessionManager.ts

```diff
@@ -419,17 +424,7 @@ class UserSessionManager implements IUserSessionManager {
         storageType === COOKIE_STORAGE
       ) {
         // remove cookie that is set from server side
```

Comment:
Apparently, the operation is feasible.
Let's add a TODO to come back to it after testing.

Reply:
Addressed